samples/django/django_user/requirements.txt

```diff
@@ -0,0 +1,3 @@
+Django==0.10.0
```

Comment:
Cool. I haven't had the chance to actually try the code yet, will update when I do.

Thanks a lot for updating the samples!